### PR TITLE
Add environment-variables article

### PR DESCRIPTION
**Pull request by mattymatny**
Wrote a cleaner more specific to our needs set of instructions for removing _JAVA_OPTIONS, included images have been cleaned and optimized as previously requested.

### CODE REVIEW

**Comment by mattymatny** [comment]
I will also need to change the incompatible software article if we decide to use this.

File: _support-articles/environment-variables.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+4\. In the Environment Variables window you should see two lists. You need to check both lists for the following listed under the Variable column: **_JAVA_OPTIONS**
+
+![](/static/images/support-articles/env-vars/delvar.png)
+
+5\. If you find that setting in either list. Select it, then press the delete button. Can then press the "Okay", or "Apply" buttons until all the windows are closed.
+
+  **NOTE:** If you did not originally add this setting on your own, make sure you have removed the [incompatible program](https://minecraftirc.net/support-articles/known-incompatible-software/) that caused it, otherwise it will re-add this setting and you will need to complete these instructions again.
+
+6\. Open the Minecraft launcher and try to run the game again.
```
Comment (urielsalis) [inline _support-articles/environment-variables.md]:
```suggestion
6\. Restart your PC  

7\. Open the Minecraft launcher and try to run the game again.
```

Comment (urielsalis) [reply _support-articles/environment-variables.md]: A logoff is usually needed, restart is easier to explain

File: _support-articles/windows/environment-variables.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+4\. In the Environment Variables window you should see two lists. You need to check both lists for the following listed under the Variable column: **_JAVA_OPTIONS**
+
+![](/static/images/support-articles/env-vars/delvar.png)
+
+5\. If you find that setting in either list. Select it, then press the delete button. Can then press the "Okay", or "Apply" buttons until all the windows are closed.
+
+  **NOTE:** If you did not originally add this setting on your own, make sure you have removed the [incompatible program](https://minecraftirc.net/support-articles/known-incompatible-software/) that caused it, otherwise it will re-add this setting and you will need to complete these instructions again.
+
+6\. Open the Minecraft launcher and try to run the game again.
```
Comment (urielsalis) [inline _support-articles/windows/environment-variables.md]:
```suggestion
6\. Restart your PC  

7\. Open the Minecraft launcher and try to run the game again.
```

**Pull request merged**